src/backend/lima.ts

```diff
@@ -1245,6 +1243,23 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
     }
   }
 
+  /**
+   * Read the given text file.
+   * @param [filePath] the path of the file to read.
+   * @param [options] Optional configuratino for reading the file.
+   * @param [options.encoding='utf-8'] The encoding to use for the result.
+   * @param [options.resolveSymlinks=true] Whether to resolve symlinks before reading.
+   */
+  async readFile(filePath: string, options?: Partial<{
+    resolveSymlinks: true,
+  }>) {
+    if (options?.resolveSymlinks ?? true) {
+      filePath = (await this.execCommand({ capture: true }, 'busybox', 'readlink', '-f', filePath)).trim();
```

Comment:
Should we always use the full path?

```suggestion
      filePath = (await this.execCommand({ capture: true }, '/bin/busybox', 'readlink', '-f', filePath)).trim();
```